src/main/java/uk/ac/ebi/tsc/tesk/util/component/KubernetesClientWrapper.java

```diff
@@ -94,6 +98,9 @@ public V1JobList listAllTaskmasterJobsForUser(String pageToken, Integer itemsPer
             // and optionally also to only those jobs, which were created bu the user
             labelSelector += "," + user.getLabelSelector();
         }
+        if(authorisationProperties.isIgnoreGroupMembership()) {
+            labelSelector += "," + new StringJoiner("=").add(LABEL_USERID_KEY).add(user.getUsername()).toString();
+        }
```

Comment:
Here formatting does not follow the standards used in the code (space after if) and I believe this code does not work for a user, who belongs to any group: the tasks outside of any groups, but still belonging to the user will not show up. The idea is that the label selector for `ignoreMembership` should not include checking groups at all.